.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+name: tests
+
+on:
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    permissions: write-all
```

Comment:
Do we really need `write all` permissions for this job?

Reply:
removed